src/cascadia/TerminalSettingsEditor/SettingContainer.cpp

```diff
@@ -189,19 +188,13 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         }
         else if (originTag == Model::OriginTag::Generated)
         {
-            const auto profileSource{ profile.Source() };
-            if (profileSource == WslGeneratorNamespace ||
-                profileSource == AzureGeneratorNamespace ||
-                profileSource == PowershellCoreGeneratorNamespace)
-            {
-                // from a dynamic profile generator
-                return {};
-            }
-            else
-            {
-                // proto-extensions
-                return hstring{ fmt::format(std::wstring_view(RS_(L"SettingContainer_OverrideMessageProtoExtension")), profileSource) };
-            }
+            // from a dynamic profile generator
+            return {};
+        }
+        else if (originTag == Model::OriginTag::Fragment)
+        {
+            // from a fragment extension
+            return hstring{ fmt::format(std::wstring_view(RS_(L"SettingContainer_OverrideMessageFragmentExtension")), profile.Source()) };

```

Comment:
prefer `{}` construction.

```suggestion
            return hstring{ fmt::format(std::wstring_view{ RS_(L"SettingContainer_OverrideMessageFragmentExtension") }, profile.Source()) };
```